mobile/src/components/uam-groups-config/uam-configure-groups.tsx

```diff
@@ -0,0 +1,165 @@
+      defaultValues: CREATE_GROUP_DEFAULT_PAYLOAD,
+      validationSchema: groupCreateClient,
+    });
+
+  const {
+    items: permissionIds,
+    handleToggle: handleTogglePermissions,
+    setItems: setDefaultPermissionIds,
+  } = useSelectedItems<number>(group?.permissionIds ?? []);
+
+  const {
+    items: userIds,
+    handleToggle: handleToggleUsers,
+    setItems: setDefaultUserIds,
+  } = useSelectedItems<number>(group?.userIds ?? []);
+
+  const paginationForUsersTable = {
+    page: usersPage,
+    setPage: handleUserPageChange,
+  };
+
+  const paginationForPermissionsTable = {
+    page: permissionsPage,
+    setPage: handlePermissionsPageChange,
+  };
+
+  const handleCreateGroup = async (): Promise<void> => {
+    await dispatch(
+      groupsCreationActions.createGroup({
+        name: control._formValues.name,
+        permissionIds,
+        userIds,
+      }),
+    ).unwrap();
+    navigation.navigate(AppScreenName.UAM);
+  };
+
+  const handleEditGroup = async (): Promise<void> => {
+    if (!group) {
+      return;
+    }
+
+    await dispatch(
+      uamGroupEditActions.editGroup({
+        id: group.id,
+        payload: {
+          name: control._formValues.name,
+          permissionIds,
+          userIds,
+        },
+      }),
+    ).unwrap();
+    navigation.navigate(AppScreenName.UAM);
+  };
+
+  const handleCancelEdit = async (): Promise<void> => {
+    dispatch(uamGroupEditActions.cancelEdit);
+    navigation.navigate(AppScreenName.UAM);
+  };
+
+  useEffect(() => {
+    dispatch(groupsCreationActions.getPermissions());
+  }, []);
+
+  useEffect(() => {
+    dispatch(
+      groupsCreationActions.getUsersForCreation({
+        page: usersPage,
+        count: PaginationDefaultValue.DEFAULT_COUNT,
+      }),
+    );
+  }, [usersPage]);
+
+  useEffect(() => {
+    if (group) {
+      setDefaultPermissionIds(group.permissionIds);
+      setDefaultUserIds(group.userIds);
+      reset({ name: group.name });
+    }
+  }, [group]);
+
+  return (
+    <View style={styles.container}>
+      <ScrollView>
+        <View style={styles.inputContainer}>
+          <Input
+            name="name"
+            placeholder="Enter group name"
+            label="Group name"
+            control={control}
+            errors={errors}
+          />
+        </View>
+
+        <Text style={styles.title}>Add users to the Group - Optional</Text>
+        <UsersTable
+          users={users}
+          onCheckboxToggle={handleToggleUsers}
+          pagination={paginationForUsersTable}
+        />
+        <Text style={styles.title}>Attach permissions policies</Text>
+        <PermissionsTable
+          permissions={permissions.items}
+          onCheckboxToggle={handleTogglePermissions}
+          pagination={paginationForPermissionsTable}
+        />
+        <View style={styles.buttonsContainer}>
+          {group && (
+            <Button label="Cancel" onPress={handleSubmit(handleCancelEdit)} />
+          )}
```

Comment:
We don't need to submit something when we cancel editing, we should return to uam screen, will add this button to the header later
```suggestion
```

Reply:
I'll remove handleSubmit, but wouldn't remove button at all for now, because onPress group in store will be set to null and it will prevent incorrect rendering of the screen, when we will have group in store and call this component in case of creating group.
I think it will be better for now, but I could be wrong

Reply:
Ok, let's leave it for now, remove handleSubmit only